examples/scheduler-policy-config.json

```diff
@@ -2,7 +2,7 @@
 "kind" : "Policy",
 "apiVersion" : "v1",
 "predicates" : [
-	{"name" : "PodFitsPorts"},
+	{"name" : "PodFitsHostPorts"},
```

Comment:
PodFitsPorts is also registered for backward compatibility. But I think I am ok to replace it to PodFitsHostPorts in example here.